templates/redis.conf.j2

```diff
@@ -765,4 +765,4 @@ hz 10
 # the file will be fsync-ed every 32 MB of data generated. This is useful
 # in order to commit the file to the disk more incrementally and avoid
 # big latency spikes.
-aof-rewrite-incremental-fsync yes
+aof-rewrite-incremental-fsync yes
```

Comment:
looks like this needs a newline added to end of file

Reply:
There should be now a new line inserted at the of the file.